qa/suites/fs/upgrade/upgraded_client/tasks/2-workload/new_ops/0-clients/kclient.yaml

```diff
@@ -0,0 +1,4 @@
+teuthology:
+  postmerge:
+    - if not is_kupstream() or syntax_version() == 'v2' then reject() end
```

Comment:
Let's add a comment explaining when/how we can remove these restrictions.

Reply:
Done. Thanks.